episodes/manage_images.md

```diff
@@ -14,23 +14,34 @@ OVERVIEW
 :::{grid-item}
 Questions
 
-- TODO 
+- What software environments are available on the hub?
+- How is containerization helpful for my work?
+- Is there a way to customise my software environment?
 
 :::
 
 :::{grid-item}
 Objectives
 
-- explain the value of reproducibility with containerized software environments
-- select pre-configured JupyterLab or RStudio image containers for their hub
-- pull a custom image container for their hub
+- Explain the value of reproducibility with containerized software environments
+- Select pre-configured JupyterLab or RStudio containers for their hub
+- Pull a custom container for their hub
 :::
 
 :::::
 
-## Exercises
+## Software for research and education
 
-::::{dropdown} Software Challenges
+Hub users more than likely require specific software packages to carry out their work. Installing software on a laptop significantly differs from installing software on a shared resource such as a hub in the cloud, largely because of different
```

Comment:
```suggestion
Hub users usually require specific software packages to carry out their work. Installing software on a laptop significantly differs from installing software on a shared resource such as a hub in the cloud, largely because of different
```